src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -170,7 +171,8 @@ export default class DefaultDeviceController
   constructor(
     private logger: Logger,
     options?: { enableWebAudio?: boolean },
-    private browserBehavior: ExtendedBrowserBehavior = new DefaultBrowserBehavior()
+    private browserBehavior: ExtendedBrowserBehavior = new DefaultBrowserBehavior(),
+    public eventController?: EventController | undefined
```

Comment:
For learning: Does this need `| undefined` as it is optional ?

Reply:
No this doesn’t, I think I put it there for testing at earlier and forgot to rid of it.